bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,416 @@
+              resources:
+                - clusterrolebindings
+                - rolebindings
+              verbs:
+                - create
+                - delete
+                - get
+                - list
+                - patch
+                - update
+                - watch
+            - apiGroups:
+                - rbac.authorization.k8s.io
+              resources:
+                - clusterroles
+                - roles
+              verbs:
+                - bind
+                - create
+                - delete
+                - escalate
+                - get
+                - list
+                - patch
+                - update
+                - watch
+            - apiGroups:
+                - view.open-cluster-management.io
+              resources:
+                - managedclusterviews
+              verbs:
+                - create
+                - delete
+                - get
+                - list
+                - patch
+                - update
+                - watch
+            - apiGroups:
+                - work.open-cluster-management.io
+              resources:
+                - manifestworks
+              verbs:
+                - create
+                - delete
+                - get
+                - list
+                - patch
+                - update
+                - watch
+          serviceAccountName: idp-mgmt-operator-controller-manager
+        - rules:
+            - apiGroups:
+                - ""
+              resources:
+                - configmaps
+              verbs:
+                - get
+                - list
+                - watch
+            - apiGroups:
+                - authorization.k8s.io
+              resources:
+                - subjectaccessreviews
+              verbs:
+                - create
+            - apiGroups:
+                - flowcontrol.apiserver.k8s.io
+              resources:
+                - prioritylevelconfigurations
+                - flowschemas
+              verbs:
+                - get
+                - list
+                - watch
+          serviceAccountName: idp-mgmt-operator-webhook-service
+      deployments:
+        - name: idp-mgmt-operator-controller-manager
+          spec:
+            replicas: 1
+            selector:
+              matchLabels:
+                control-plane: controller-manager
+            strategy: {}
+            template:
+              metadata:
+                labels:
+                  control-plane: controller-manager
+              spec:
+                containers:
+                  - args:
+                      - manager
+                      - --enable-leader-election
+                      - --
+                      - -v 5
+                    env:
+                      - name: RELATED_IMAGE_DEX_OPERATOR
+                        value: quay.io/identitatem/dex-operator@sha256:b0b7c7f26f966782bb897378d4ccf26759ab2faa69a72375fbc1c65706da6b49
```

Comment:
This needs to be updated to 2b13763d838ee90427ab23ad44df53e5dabaf37fa575ba548bd61efecc7bba82 based on PR 75 (not merged yet).  I assume this is via automated make step

Reply:
Merged and updated!